tutorials/nservicebus-step-by-step/3-multiple-endpoints/tutorial.md

```diff
@@ -81,9 +81,11 @@ Let's split apart the endpoint we created in the previous lesson. We'll reconfig
 First, let's create a project for our new endpoint.
 
  1. Create a new **Console Application** project named **Sales**.
- 1. In the same directory as the **Sales** project, add the NServiceBus NuGet package using the .NET CLI:
+ 1. In the same directory as the **Sales** project, add the following NuGet packages using the .NET CLI:
       ```
+    dotnet add package Microsoft.Extensions.Hosting
```

Comment:
Microsoft.Extensions.Hosting is a transitive dependency of NServiceBus.Extensions.Hosting so it doesn't need to be added explicity. Did you have issues with compilation?
```suggestion
```